docs/jwst/tweakreg/README.rst

```diff
@@ -165,17 +166,21 @@ telescope pointing will be identical in all these images and it is assumed
 that the relative positions of (e.g., NIRCam) detectors do not change.
 Identification of images that belong to the same "exposure" and therefore
 can be grouped together is based on several attributes described in
-`~jwst.datamodels.ModelContainer`. This grouping is performed automatically
+`~jwst.datamodels.ModelLibrary`. This grouping is performed automatically
 in the ``tweakreg`` step using the
-`~jwst.datamodels.ModelContainer.models_grouped` property, which assigns
+`~jwst.datamodels.ModelLibrary.models_grouped` property, which assigns
 a group ID to each input image model in ``meta.group_id``.
+## FIXME: The ModelLibrary does not have a models_grouped property.
```

Comment:
Hopefully one of the reviewers of this PR can provide some clarity here?

Reply:
This statement is out-dated since https://github.com/spacetelescope/jwst/pull/8424 The grouping is the same but the step no longer (directly) uses `models_grouped`.